src/AutoBuild.jl

```diff
@@ -76,6 +76,12 @@ download, build and package the tarballs, generating a `build.jl` file when
 appropriate.  Note that `ARGS` should be the top-level Julia `ARGS` command-
 line arguments object.  This function does some rudimentary parsing of the
 `ARGS`, call it with `--help` in the `ARGS` to see what it can do.
+
+The `kwargs` are passed on to [`autobuild`](@ref), see there for a list of
+supported ones. In addition, the keyword argument `init_block` may be set to
+a string containing Julia code; if present, this code will be inserted into
+the initialization path of the generated JLL package. This can for example be
+used to invoke an initialization API of a shared library.
```

Comment:
This is where you should mention that `init_block` is experimental and may be removed in a future release.

Reply:
Done via `!!! note`, let me know if you prefer something else.